lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -88,6 +75,7 @@ public function get_items( $request ) {
 		$args                   = array();
 		$args['paged']          = $request['page'];
 		$args['posts_per_page'] = $request['per_page'];
+		$args['post_parent']    = $request['parent'];
```

Comment:
One thing that's a little dodgy is that _any_ controller extending this controller could use `parent` in the request, because we pass all arguments present in the request.